src/openms/source/FORMAT/HANDLERS/XMLHandler.cpp

```diff
@@ -204,27 +204,14 @@ namespace OpenMS::Internal
       DataValue cv_value = value;
 
       // Abort on unknown terms
-      if (!cv.exists(accession))
+      try
```

Comment:
threads safe?

Reply:
the exception is caught below, if one should occur.